Generator/AssemblyInfo.fs

```diff
@@ -0,0 +1,17 @@
+﻿// Auto-Generated by FAKE; do not edit
```

Comment:
This was added by FAKE - do we need one in the repo by default for people who build inside VS instead of from the script?

Reply:
Yes, it's correct to add it to the repo